spec/factories/users.rb

```diff
@@ -1,4 +1,10 @@
 FactoryBot.define do
   factory :user do
-  end
+    email {Faker::Internet.email}
+    password {Faker::Internet.password}
+    first_name {Faker::Name.first_name}
+    last_name {Faker::Name.last_name}
+    cohort {"1911"}
+    status {1}
+    end
```

Comment:
Cool factorybot setup. Good job adhering to the model/db validations we set up in each model.

Reply:
Thanks I read through Faker documentation unfortunatley it will not work with questions, comments, or answers :(